client/components/data/query-preferences/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingPreferences } from 'state/preferences/selectors';
+import { fetchPreferences } from 'state/preferences/actions';
+
+class QueryPreferences extends Component {
+
+	componentWillMount() {
+		if ( ! this.props.isFetchingPreferences ) {
+			this.props.fetchPreferences();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+QueryPreferences.propTypes = {
+	requestingSites: PropTypes.bool,
+	fetchPreferences: PropTypes.func
+};
+
+QueryPreferences.defaultProps = {
+	fetchPreferences: () => {}
```

Comment:
At least in this case (without the component being its own export), there's no chance that `fetchPreferences` would ever not be defined as a prop, so it may not be necessary to define the default.